### PR TITLE
Remove dist files, ignore them, add action to release package instead

### PR DESCRIPTION
**Pull request by ScopeyNZ**
With this, the workflow to publish the UI library becomes:

- Navigate to the actions in GitHub 
 
image

- Click the on the publish workflow, and then click the "run workflow" button

image

- Enter the new version

image

- Hit the run button

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Enter a valid semver version'
+        required: true
```
Comment (robbieaverill) [inline .github/workflows/publish.yml:8]: What does this do exactly?

**Review by robbieaverill — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (ScopeyNZ) [reply .github/workflows/publish.yml:8]: Soz, was just updating the OP with images. Raising PRs from the command line is convenient when you don't want to add images :P

**Comment by ScopeyNZ** [comment]
Actually I've forgotten to specify the NPM token here. One moment

File: .github/workflows/publish.yml (hunk 1 above)
Comment (robbieaverill) [reply .github/workflows/publish.yml:8]: Nice, looks good

**Pull request merged**